playbooks/webapp/README.md

```diff
@@ -126,7 +126,7 @@ To delete the webapp:
 * **rds_instance_class** (str): DB instance class for the RDS instance. Default: `db.m6g.large`
 * **rds_instance_name** (str): Name for the database. Default: `mysampledb123`
 * **rds_engine** (str): Engine to use for the database. Default: `postgres`
-* **rds_engine_version** (str): Version number of the database engine to use. Default: `"14.8"`
+* **rds_engine_version** (str): Version number of the database engine to use. Default: `"16.2"`
```

Comment:
I fail to see why this modification would constitute a breaking change. AWS has eliminated the 14.8 version. Users can still specify the desired version number as they did previously. If not specified, the default value will now be 16.2 instead of 14.8. This adjustment should have no impact on users of this role. Does this make sense to you?